cmd/soroban-cli/src/commands/network/container/start.rs

```diff
@@ -105,18 +108,42 @@ async fn run_docker_command(cmd: &Cmd) -> Result<(), Error> {
         )
         .await?;
     println!("✅ Container started: {container_name}");
-    let stop_message = format!(
-        "ℹ️  To stop this container run: stellar network stop {network} {additional_flags}",
-        network = &cmd.network,
-        additional_flags = if cmd.docker_host.is_some() {
-            format!("--docker-host {}", cmd.docker_host.as_ref().unwrap())
-        } else {
-            String::new()
-        }
+    print_log_message(cmd);
+    print_stop_message(cmd);
+    Ok(())
+}
+
+fn print_log_message(cmd: &Cmd) {
+    let log_message = format!(
+        "ℹ️ To see the logs for this container run: stellar network container logs {arg} {additional_flags}",
+        arg = cmd.container_args.container_name.as_ref().map_or_else(
+            || cmd.network.to_string(),
+            |container_name| format!("--container-name {container_name}")
+        ),
+        additional_flags = cmd.container_args.docker_host.as_ref().map_or_else(
+            String::new,
+            |docker_host| format!("--docker-host {docker_host}")
+        )
```

Comment:
Perhaps you could add two methods to the `impl Args` that get the arg strings.

Reply:
Great idea! Just pushed an update